src/renderer/utils/InstallManager.tsx

```diff
@@ -256,7 +258,64 @@ export class InstallManager {
             }
         }
 
-        const destDir = Directories.inCommunity(addon.targetDirectory);
+        // Find incompatible add-ons
+        // This iterates through the first level of folders of the  MSFS Community folder looking for the manifest.json
+        // file. It compares the manifest.json file content with the configured incompatible add-ons (data.ts) and if it
+        // finds one or more matches, it will issue a warning.
+        console.log('Searching incompatible add-ons');
+        const manifestFileName = 'manifest.json';
+        const comDir = Directories.communityLocation();
+        const incompatibleAddons: AddonIncompatibleAddon[] = [];
+        try {
+            const addonFolders = fs.readdirSync(comDir);
```

Comment:
**suggestion:** extract this logic to some other file

Reply:
That would be inconsistent - dependency check is also inline in the file.

Reply:
Nothing else here is doing complex filesystem work on things unrelated to the install itself. Hence why it has to go into another file. This is done the same way for external apps, background services, etc. and their related checks.

Reply:
Ok - I can extract this into a separate class if you think it is necessary. 
Will not be able to really work on this before Friday though.